projects/file-parsing/types-converting/json-struct.go

```diff
@@ -0,0 +1,33 @@
+package filesConverting
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type Score struct {
```

Comment:
I would probably put this struct in a file called `common.go` or `types.go` or something in this package - it's not specifically json-related :)